app/modules/booking/service.server.ts

```diff
@@ -1536,3 +1536,36 @@ export async function addScannedAssetsToBooking({
     });
   }
 }
+
+export async function getExistingBookingDetails(bookingId: string) {
+  try {
+    const booking = await db.booking.findFirst({
```

Comment:
Usually we use `findUniqueOrThrow` as this automatically handles 404s. Is there some specific reason to not use it here.